app/src/main/java/com/swent/assos/model/data/News.kt

```diff
@@ -7,7 +7,7 @@ import java.time.LocalDateTime
 data class News(
     val id: String = "",
     val title: String = "",
-    val associationId: String = "",
+    var associationId: String = "",
```

Comment:
Could you please help me understand the need to change it's type to var.

Reply:
Android studio wouldn't let me leave this as a val as the variable. I just checked and the code still compiles with the variable as a val so I'll revert my change.